atom/kite/lib/kite.js

```diff
@@ -332,9 +333,93 @@ var KiteIncoming = {
     }
     return null;
   },
-
 };
 
+var CompletionsProvider = {
```

Comment:
can we split this out to it's own module and then require it instead of defining it inline?